src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/models/SubmissionTest.java

```diff
@@ -21,46 +23,48 @@
     private static final String CANCEL = "CANCEL";
     private static final String ERROR = "ERROR";
     private static final String TYPE = "TYPE";
-    private static final String SUBTYPE = "SUBTYPE";
-    private static final String URL = "http://doc.com";
-    private static final String HEADER = "HEADER";
     private static final String FIRST_NAME = "firstName";
     private static final String LAST_NAME = "lastName";
     private static final String MIDDLE_NAME = "middleName";
     private static final String EMAIL = "email";
-    private static final String APPLICATION_TYPE = "ApplicationType";
-    private static final String COURT_LOCATION = "CourtLocation";
-    private static final String COURT_LEVEL = "CourtLevel";
-    private static final String COURT_DIVISION = "CourtDivision";
-    private static final String COURT_CLASS = "CourtClass";
-    private static final String PARTICIPATION_CLASS = "ParticipationClass";
-    private static final String INDIGENOUS_STATUS = "IndigenousStatus";
-    private static final String DOCUMENT_TYPE = "DocumentType";
-    private static final String COURT_FILE_NUMBER = "CourtFileNumber";
+    private static final String DISPLAYNAME = "DISPLAYNAME";
+    private static final String DIVISION = "DIVISION";
+    private static final String FILENUMBER = "FILENUMBER";
```

Comment:
in the past ive seen stuff like this named as:
```suggestion
    private static final String FILENUMBER = "FILE_NUMBER";
```
so wondering why we arent using that here